lerobot/scripts/control_robot.py

```diff
@@ -218,6 +219,7 @@ def record(
     policy = None
     device = None
     use_amp = None
+    features = {"reward": {"dtype": "int64", "shape": (1,), "names": None}} if assign_rewards else None
```

Comment:
is "reward" the right naming or should it be "next.reward" ?

Reply:
It should be 'next.reward' :)

Reply:
next.reward corresponds to the reward you will get if you take the given action in the given state at t. So it scores the state at t+1.
Does it make sense? :')